scanner/e2etests/config_test.go

```diff
@@ -0,0 +1,83 @@
+package e2etests
+
+import (
+	"context"
+	"crypto/sha256"
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/golang/mock/gomock"
```

Comment:
let's use `uber-go/mock` instead, as it's still maintained

Reply:
I tried with uber-go/mock but mockIndexer of Claircore uses golang/mock
```
ctrl := gomock.NewController(t)
s := mockIndexer.NewMockStore(ctrl)
```